docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -774,7 +775,7 @@
               register: new_reboot
               when: waiting_after_new_reboot.failed
 
-            when: waiting_after_reboot.failed
+            when: waiting_after_reboot.failed or waiting_after_new_reboot.unreachable
```

Comment:
Further, I believe the previous change regarding the "ignore_unreachable" of waiting_after_reboot was not properly tested.

Reply:
It was tested after changing the condition to when: waiting_after_reboot.failed or waiting_after_reboot.unreachable